charts/spire/charts/spire-server/README.md

```diff
@@ -233,6 +235,7 @@ In order to run Tornjak with simple HTTP Connection only, make sure you don't cr
 | `controllerManager.identities.namespaceSelector`                 | Selector for namespacs to issue identity                                                                                                                                                                                                                                                                                                                             | `{}`                                                                                           |
 | `controllerManager.identities.dnsNameTemplates`                  | DNS name template for issued identities                                                                                                                                                                                                                                                                                                                              | `[]`                                                                                           |
 | `controllerManager.identities.federatesWith`                     | Other Spire server URLs for identity federation                                                                                                                                                                                                                                                                                                                      | `[]`                                                                                           |
+| `controllerManager.identities.downstream`                        | set if this is a root server                                                                                                                                                                                                                                                                                                                                         | `false`                                                                                        |
```

Comment:
Shouldn't the default be true, so it works out of the box for the simple usecase where you just have a single server?

Reply:
Downstream means that the cluster spire identity will be a downstream of this instance and therefore given intermediate CA's. So, only should be set on root spire instances.

Reply:
That is my point. If I just have a simple setup, I should not require to set any value.

I think it is better to set it only when running downstream servers to keep the simple setup with proper defaults.

Reply:
thats my point. in a default setup, it is correct being false. you only need to set it as true, if your setting up a nested deployment and the release is the root one.